docs/datamodel/scalars/bytes.rst

```diff
@@ -28,3 +28,12 @@ Bytes
 
         db> SELECT contains(b'qwerty', b'42');
         {false}
+
+
+See Also
+--------
+
+Scalar type
+:ref:`SDL <ref_eql_sdl_scalars>`,
+:ref:`DDL <ref_eql_ddl_scalars>`,
+and :ref:`introspection <ref_eql_introspection_scalar_types>`.
```

Comment:
I'd also link relevant operators and functions from each data type page

Reply:
I've linked the sections from funcops.